### ISSUE TITLE
High write I/O and cpu consumption when restart racecar process

### ISSUE THREAD

**Issue by liuzxc** [zendesk/racecar#100]
<img width="673" alt="2018-12-07 8 03 56" src="https://user-images.githubusercontent.com/1954295/49646836-6055be00-fa5b-11e8-88d9-e8fa15fb29fc.png">

![2018-12-07 8 13 19](https://user-images.githubusercontent.com/1954295/49647168-93e51800-fa5c-11e8-8255-5e9ebda923a4.png)

```
class TestConsumer < Racecar::Consumer
  subscribes_to "test"

  def process(message)
    puts "Received message: #{message.value}"
  end
end
```

I am using the default configration.

ruby version: 2.4.2
racecar version: 0.3.7
kafka version: 0.10.0.0

**Comment by liuzxc** [comment]
I found the problem was that loading the application was too slow.

```
      if defined?(Rails)
        $stderr.puts "=> Detected Rails, booting application..."

        require "./config/environment"
```
load rails application will spend 70 seconds.

**Comment by latortuga** [comment]
This happened to me and it was bootsnap. Are you using it? You can fix the issue as I described in #95

**Comment by liuzxc** [comment]
@latortuga yes, I am using the bootsnap, when I remove it, boot up is very fast!
